uploadNeuroDB/NeuroDB/MRIProcessingUtility.pm

```diff
@@ -1242,6 +1242,80 @@ sub computeSNR {
 }
 
 ################################################################
+##########  Order Imaging Modalities By Acq ####################
+################################################################
+
+sub orderModalitiesByAcq {
+
+    my $this = shift;
+    my ($row, $file, $acqProtID, $dataArr, $cmd, $message, $sth);
+    my ($tarchiveID, $tarchive_srcloc)= @_;
+    my $upload_id = getUploadIDUsingTarchiveSrcLoc($tarchive_srcloc);
+
+    my $query = "SELECT DISTINCT f.AcquisitionProtocolID ".
+		"from files f ";
+    my $where = "WHERE f.TarchiveSource=?";   
+    $query = $query . $where;
```

Comment:
The $where variable is not useful. I would write the whole query directly in the $query variable.
Also, FROM was not capitalized here (not crucial but to stay consistent)